contribs/gmf/apps/service.js

```diff
@@ -0,0 +1,41 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2016-2021 Camptocamp SA
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of
+// this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to
+// use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+// the Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+/**
+ * Application entry point.
+ *
+ * This file includes `import`'s for all the components/directives used
+ * by the HTML page and the controller to provide the configuration.
+ */
+
+export const Service = {
+    /** @type {angular.IHttpService} */
+    http: null,
+
+    /** @type {import("gmf/authentication/Service.js").AuthenticationService} */
+    auth: null,
+
+    /** @type {import('gmf/authentication/Service.js').User} */
+    user: null,
+
+    /** @type {import("ngeo/message/Notification.js").MessageNotification} */
+    notification: null,
+};
```

Comment:
For me, this file should be in the folder src...

Reply:
For me too, also because we have to build the gmf-apps (as it's in apps) to run the (ongoing) storybook example

Reply:
As explained in the meeting, I will use a more usefull name but not put it in auth folder as it will be the name of the current authentication/service.js that is in angular. I'm going to use root of src/ and import it as 'AngularServices' in the files.